README.md

```diff
@@ -32,106 +49,31 @@ You also need to inherit your app theme from Backpack:
 
-## Components
-
-* [Badge](docs/view/Badge/README.md)
-* [Bottom Nav](docs/view/BottomNav/README.md)
-* [Bottom Sheet](docs/view/BottomSheet/README.md)
-* [Button](docs/view/Button/README.md)
-* [Card](docs/view/Card/README.md)
-* [Calendar](docs/view/Calendar/README.md) (Deprecated)
-* [Calendar2](docs/view/Calendar2/README.md)
-* [Checkbox](docs/view/Checkbox/README.md)
-* [Chip](docs/view/Chip/README.md)
-* [Dialog](docs/view/Dialog/README.md)
-* [Horizontal Nav](docs/view/HorizontalNav/README.md)
-* [Floating Action Button](docs/view/FloatingActionButton/README.md)
-* [Interactive Star Rating](docs/view/InteractiveStarRating/README.md)
-* [Map Markers](docs/view/Maps/README.md)
-* [Nav Bar](docs/view/NavBar/README.md)
-* [Nudger](docs/view/Nudger/README.md)
-* [Panel](docs/view/Panel/README.md)
-* [RadioButton](docs/view/RadioButton/README.md)
-* [Overlay](docs/view/Overlay/README.md)
-* [Rating](docs/view/Rating/README.md)
-* [Slider](docs/view/Slider/README.md)
-* [Snackbar](docs/view/Snackbar/README.md)
-* [Spinner](docs/view/Spinner/README.md)
-* [Star Rating](docs/view/StarRating/README.md)
-* [Switch](docs/view/Switch/README.md)
-* [Text](docs/view/Text/README.md)
-* [Text Field](docs/view/TextField/README.md)
-* [Text Spans](docs/view/TextSpans/README.md)
-* [Toast](docs/view/Toast/README.md)
-
-## Usage
-
-### Radii
-
-The Backpack radii tokens are available as [dimension resource](Backpack/src/main/res/values/backpack.radii.xml).
-
-The supported tokens are
-
-+ `bpkBorderRadiusSm`
-+ `bpkBorderRadiusPill`
-
-### Elevation
-
-The Backpack elevation tokens are available as [dimension resource](Backpack/src/main/res/values/backpack.elevation.xml).
-
-The supported tokens are
-
-+ `bpkElevationXs`
-+ `bpkElevationSm`
-+ `bpkElevationBase`
-+ `bpkElevationLg`
-+ `bpkElevationXl`
-
-### Text Styles
-
-The Backpack text styles are available as [style resources](Backpack/src/main/res/values/backpack.text.xml).
-
-The supported styles are
-
-+ `bpkTextXs`
-+ `bpkTextXsEmphasized`
-+ `bpkTextSm`
-+ `bpkTextSmEmphasized`
-+ `bpkTextBase`
-+ `bpkTextBaseEmphasized`
-+ `bpkTextLg`
-+ `bpkTextLgEmphasized`
-+ `bpkTextXl`
-+ `bpkTextXlEmphasized`
-+ `bpkTextXl`
-+ `bpkTextXlEmphasized`
-+ `bpkTextXxl`
-+ `bpkTextXxlEmphasized`
-
-### Color
-```xml
-<TextView
-  android:text="This is Backpack Blue 500!"
-  android:textColor="@color/bpkSkyBlue" />
-```
+You can find the list of the available components, as well as the code samples and the screenshots [here](https://backpack.github.io/components/text?platform=compose).
 
-```kotlin
-R.color.bpkSkyBlue
+The design system tokens are located in a `net.skyscanner.backpack.compose.tokens` package.
+For instance, you can refer to color using following syntax: `BpkColor.SkyBlue`.
+
+### Installation
+
+Add the following dependency to your `build.gradle` (in your app module) in the `dependencies` block:
+
+```gradle
+implementation 'net.skyscanner.backpack:backpack-compose:1.0.0'
```

Comment:
Not perfect – we don't have any release yet, and there's no a version autoupdate when making the release. However the version autoupdate can be done as a part of a Compose release ticket